assets/scss/3-components/1-carousel.scss

```diff
@@ -0,0 +1,69 @@
+.carousel-list {
+  @include flex(row, flex-start, flex-start, wrap);
+}
+
+.carousel-list__item {
+  flex: 1 auto;
+  position: relative;
+
+  &::before {
+    content: "";
+    width: 100%;
+    height: 100%;
+    background: #000;
+    opacity: 0.2;
+    position: absolute;
+    top: 0;
+    left: 0;
+  }
+}
+
+.carousel-list__image {
+  img {
+    width: 100%;
+    display: block;
+  }
+}
+
+.carousel-list__info {
+  position: absolute;
+  z-index: 2;
+  color: $white;
+  left: 50%;
+  top: 50%;
+  transform: translate3d(-50%, -50%, 0);
+  text-align: center;
```

Comment:
Consider if it's possible to center an element without using `transform` (maybe add an extra wrapper or something). It might make the text appear blurry on some browsers/devices ;)